functions/go/fix/fixpkg/fix_test.go

```diff
@@ -0,0 +1,33 @@
+package fixpkg
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/kyaml/copyutil"
+	"sigs.k8s.io/kustomize/kyaml/kio"
+)
+
+func TestFix(t *testing.T) {
```

Comment:
Should we have a test that cover nested packages?

Reply:
Will add more tests in coming PRs. Want to keep this smaller.